docs/extras/advanced-targeting/self-targeting.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Simple Self-Targeting
+sidebar_position: 2
+---
+
+Self-Targeting is a new DevCycle QA feature that allows you to easily serve yourself different variations of a feature without having to modify targeting rules in each environment. To leverage this feature, you must set up your DevCycle Identity by associating an SDK User ID to your DevCycle user profile. 
```

Comment:
maybe call it a `QA and debugging` feature? We don't want to limit use-cases to just QA, developers would use this too